client/post-editor/media-modal/index.jsx

```diff
@@ -473,7 +441,7 @@ export default connect(
 		view: getMediaModalView( state ),
 		// [TODO]: Migrate toward dropping incoming site prop, accepting only
 		// siteId and forcing descendant components to access via state
-		site: site || getSelectedSite( state, siteId )
+		site: site || getSite( state, siteId )
```

Comment:
>Can we drop the `siteId` prop as well and use `getSelectedSite` ?

Depends on how we want the component to be rendered. In this case I'd assumed being able to explicitly control the site by prop instead of implicitly assuming a currently selected site. I think this would be more appropriate for a general-use block component.